### PR TITLE
Fix: kafka default max partitions number bug

### PR DESCRIPTION
**Pull request by liortct**
Signed-off-by: liortct <liortct@gmail.com>

There is an [open bug](https://github.com/kedacore/keda/issues/2888) in Kafka scaler, I added a documentation about this bug.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.6/scalers/apache-kafka.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ go_file = "kafka_scaler"
 > - By default, the number of replicas will not exceed:
 >   - The number of partitions on a topic when a topic is specified;
 >   - The number of partitions of *all topics* in the consumer group when no topic is specified;
+>   - 100, the default value of the ScaledObject;
```
Comment (JorTurFer) [inline content/docs/2.6/scalers/apache-kafka.md:14]: Same as the previous comment:
>   - `maxReplicaCount`

File: content/docs/2.7/scalers/apache-kafka.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ go_file = "kafka_scaler"
 > - By default, the number of replicas will not exceed:
 >   - The number of partitions on a topic when a topic is specified;
 >   - The number of partitions of *all topics* in the consumer group when no topic is specified;
+>   - 100, the default value of the ScaledObject;
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/apache-kafka.md:14]: same as above

**Review by JorTurFer — COMMENTED** [review]
Hi!
I have left some comments inline

**Comment by liortct** [comment]
> Hi! I have left some comments inline

Fixed

File: content/docs/2.6/scalers/apache-kafka.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ go_file = "kafka_scaler"
 > - By default, the number of replicas will not exceed:
 >   - The number of partitions on a topic when a topic is specified;
 >   - The number of partitions of *all topics* in the consumer group when no topic is specified;
+>   - maxReplicaCount, the default value of the ScaledObject;
```
Comment (zroubalik) [inline content/docs/2.6/scalers/apache-kafka.md:14]:
```suggestion
>   - `maxReplicaCount` specified in `ScaledObject`/`ScaledJob`. If not specifed, then the default value of `maxReplicaCount` is used;
```

what about something like this?

Comment (liortct) [reply content/docs/2.6/scalers/apache-kafka.md]: I liked your comment :)
Fixed

**Review by zroubalik — APPROVED** [review]

**Comment by liortct** [comment]
Can someone merge this PR?
@zroubalik @JorTurFer

**Pull request merged**